configs/promtail/promtail-config.yml

```diff
@@ -0,0 +1,38 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://loki:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: system
+  static_configs:
+  - targets:
+      - localhost
+    labels:
+      job: flasklogs
+      __path__: /var/log/flask.log
```

Comment:
I am not sure this is needed?

Reply:
copy/paste issue from previous lab. Not needed